ibis/bigquery/tests/test_client.py

```diff
@@ -546,6 +546,14 @@ def test_client_sql_query(client):
     tm.assert_frame_equal(result, expected)
 
 
+def test_trailing_time_window_order_by(alltypes):
+    # trailing_time_window must take time column in order_by
+    with pytest.raises(com.IbisInputError):
+        w = ibis.trailing_time_window(preceding=10, order_by='float_col')
+        expr = alltypes.int_col.mean().over(w)
+        expr.execute()
```

Comment:
Can you put only the line that causes the error underneath the `pytest.raises` context manager? Otherwise, the test isn't clear about what's being tested since if any of those lines raises `com.IbisInputError` exception then pytest calls it a pass.